videointelligence/annotate/object_tracking_gcs_test.go

```diff
@@ -18,21 +18,25 @@ import (
 	"bytes"
 	"strings"
 	"testing"
+	"time"
 
 	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
 )
 
 func TestObjectTrackingGCS(t *testing.T) {
-	testutil.SystemTest(t)
+	testutil.EndToEndTest(t)
 
 	gcsURI := "gs://cloud-samples-data/video/cat.mp4"
 
-	var buf bytes.Buffer
-	if err := objectTrackingGCS(&buf, gcsURI); err != nil {
-		t.Fatal(err)
-	}
+	testutil.Retry(t, 10, 20*time.Second, func(r *testutil.R) {
```

Comment:
20s sleep between retries seems long, but fine if that's what you think makes sense to avoid the quota issue.